lib/install/config/webpack/development.server.js

```diff
@@ -1,18 +1,33 @@
 // Note: You must restart bin/webpack-dev-server for changes to take effect
 
-const { resolve } = require('path')
 const merge = require('webpack-merge')
+const ManifestPlugin = require('webpack-manifest-plugin')
 const devConfig = require('./development.js')
-const { devServer, publicPath, paths } = require('./configuration.js')
+const { devServer, paths, output, formatPublicPath } = require('./configuration.js')
+
+const { host, port } = devServer
+const contentBase = output.path
+const publicPath = formatPublicPath(`http://${host}:${port}`, paths.output)
+
+// Remove ManifestPlugin so we can replace it with a new one
+devConfig.plugins = devConfig.plugins.filter(plugin => plugin.constructor.name !== 'ManifestPlugin')
```

Comment:
If we consider using `ASSET_HOST` env in development then I guess we don't need any of the changes here :)